### PR TITLE
Included default download directory and added functionality to remove created APIs after the test

### PR DESCRIPTION
**Pull request by thilinicooray**

### CODE REVIEW

File: modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java [hunk 1]
```diff
@@ -60,11 +54,20 @@
     private ServerConfigurationManager configManagerDB, configManagerMasterDatasource, configManagerReg;
     private WebDriver driver;
     private WebDriverWait wait;
+    private final String SUPER_TENANT_DOMAIN_NAME = "carbon.super";
+    private final String SUPER_TENANT_USERNAME = "admin";
+    private final String SUPER_TENANT_PASSWORD = "admin";
+    private final String TENANT_DOMAIN_NAME = "wso2.com";
```
Comment (sanjeewa-malalgoda) [inline modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java]: You may use multi tenant utils to get super tenant username domain etc. No need to hard code them here.

File: modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java [hunk 2]
```diff
@@ -180,7 +185,7 @@ public void createAPIWithWSDLEndpoint(String apiName, String username, String pa
         driver.findElement(By.xpath("(//input[@name='create-api'])[2]")).click();
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("startFromExistingSOAPEndpoint")));
         driver.findElement(By.id("wsdl-url")).clear();
-        driver.findElement(By.id("wsdl-url")).sendKeys("http://www.webservicex.net/geoipservice.asmx?WSDL");
+        driver.findElement(By.id("wsdl-url")).sendKeys("http://www.webservicex.com/globalweather.asmx?WSDL");
         driver.findElement(By.id("startFromExistingSOAPEndpoint")).click();
```
Comment (sanjeewa-malalgoda) [inline modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java]: This API is no longer available to use freely. So we may need different endpoint for this.

**Comment by thilinicooray** [comment]
Since this test case aims at downloadability of a WSDL file and we are not invoking its actual endpoint, using a freely available WSDL URL such as http://www.webservicex.com/globalweather.asmx?WSDL will be sufficient IMO.

File: modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java (hunk 1 above)
Comment (thilinicooray) [inline modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java]: Fixed in PR #460

**Pull request merged**